components/member_list_team/member_list_team.tsx

```diff
@@ -58,14 +59,11 @@ export default class MemberListTeam extends React.Component<Props, State> {
         };
     }
 
-    componentDidMount() {
-        this.props.actions.loadProfilesAndTeamMembers(0, Constants.PROFILE_CHUNK_SIZE, this.props.currentTeamId).then(({data}) => {
-            if (data) {
-                this.loadComplete();
-            }
-        });
-
+    async componentDidMount() {
+        await this.props.actions.loadProfilesAndTeamMembers(0, Constants.PROFILE_CHUNK_SIZE, this.props.currentTeamId);
+        await this.props.actions.getTeamMembers(this.props.currentTeamId);
         this.props.actions.getTeamStats(this.props.currentTeamId);
```

Comment:
Also, this could happen before the await so that it's started without waiting for the other 2 requests